### DISCUSSION TITLE
How do I add custom css properties?

### DISCUSSION THREAD

**Discussion by abhishek-junghare** [home-sweet-gnome/dash-to-panel#2195] [Q&A] [answered]
I want to style this window preview box, but I couldn't find any relative .css file in the extension directory. I think, it's been handled in those .ui files? But, how do I style these elements with css instead?

What I want is, to set custom border-radius of these previews box.

![Screenshot from 2024-10-14 04-12-41](https://github.com/user-attachments/assets/2204c922-bec0-47ea-8634-d70424e5fc9b)

**Comment by charlesg99** [comment, marked as answer]
Have a look here: https://github.com/home-sweet-gnome/dash-to-panel/issues/726
